pkgs/shells/marcel/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, dill
+, psutil
+, makeWrapper
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "marcel";
+  version = "0.17.0-unstable-2023-11-11";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "geophile";
+    repo = "marcel";
+    rev = "ed96ef5da31e7c099e22f11911af28eb4425d4d5";
+    hash = "sha256-503JqQb29Uwd3rm2r49accFl3pWjaX/BNziO4iZJmJ4=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    dill
+    psutil
+  ];
+
+  doCheck = false;
+
+  patches = [ ./changes.patch ];
+
+  postInstall = ''
+    cat > $out/bin/marcel-start <<EOF
+    #!/usr/bin/env sh
+    exec python3 -m marcel.main "\$@"
+    EOF
+    chmod +x $out/bin/marcel-start
+    wrapProgram $out/bin/marcel-start \
+    --prefix PYTHONPATH : $out/${python3.sitePackages}
+    mv $out/bin/marcel-start $out/bin/marcel
```

Comment:
why create it in marcel-start just to move it later?

Reply:
my mistake. can write over the generated file.